StandardLibrary/test/TestComputeSetToOrderedSequenceUInt8Less.dfy

```diff
@@ -82,12 +84,15 @@ module TestComputeSetToOrderedSequenceUInt8Less {
   }
 
   method {:test} TestSetToOrderedSequenceManyItems() {
+    var time := Time.GetAbsoluteTime();
```

Comment:
Is this leftover from your per-language performance test or do you want to commit this? (I'm fine either way, esp. since there are still egregious issues)

Reply:
Yes, I see no downside to sprinkling these into our tests here and there and my plan is to do just that.